v1/controllers/product.js

```diff
@@ -0,0 +1,340 @@
+const Models = require("../../models");
+const universal = require("../../utils");
+const CODES = require("../../constants").Codes;
+const MESSAGES = require("../../constants").Messages;
+const config = require("config");
+const USER_TYPES = Object.values(config.get("USER_TYPES"));
+const ObjectId = require("mongoose").Types.ObjectId;
+const csvParser = require('csv-parser');
+const fs = require('fs');
+const path = require('path');
+const mongoose = require('mongoose');
+
+module.exports = {
+  create: async (req, res, next) => {
+    try {
+      // Check if manufacturer exists and is active
+      const manufacturer = await Models.User.findOne({
+        _id: req.body.manufacturer,
+        type: { $in: ["MANUFACTURER_ADMIN"] },
```

Comment:
if user type comparison is only single then do direct comparison no any need to use $in